src/shogun/machine/BaggingMachine.cpp

```diff
@@ -155,26 +137,25 @@ bool BaggingMachine::train_machine(std::shared_ptr<Features> data)
 	random::fill_array(rnd_indicies, 0, m_bag_size - 1, m_prng);
 
 	auto pb = SG_PROGRESS(range(m_num_bags));
-#pragma omp parallel for
+//#pragma omp parallel for
```

Comment:
does this not work?

Reply:
When add this line, there is segmentation fault in `random_forest_regression.sg.in`, but this segmentation fault don't occur every time.

Reply:
Hmm, was this the case before you started refactoring the class?